### ISSUE TITLE
node-sass package version too low

### ISSUE THREAD

**Issue by neo** [Jam3/generator-jam3#329]
we might want to update the node-sass version in the `package.json`, it's failing on some newer node environments.
if we agree to this, I could do that. I feel like there're also a couple other packages that might need an update.

**Comment by njam3** [comment]
Agreed, it may be a good idea to do audit of all the other packages as well, see what we can update without breaking anything. Some of them have been in there for a long time.

**Comment by namniak** [comment]
dev dependencies would be updated here https://github.com/Jam3/generator-jam3/pull/319/files#diff-56eac0ea11a7d5434555fcf7ddc26ee4 but not dev deps

**Comment by neo** [comment]
ok, I guess I will test out some of the devDeps, and upgrade the non-breaking ones.
